queries.yaml

```diff
@@ -1065,7 +1065,7 @@ post_process:
       end_zoom: 11
       items_matching: { kind: locality }
       max_items: 1
-      grid_width: 6
+      grid_width: 48
```

Comment:
Try `24` (which is 4 times original value of 6)

Reply:
Resolved in 50bb8982.